justfile

```diff
@@ -85,8 +85,12 @@ install-toolchain:
 	rustup component add llvm-tools-preview --toolchain "$(cat rust-toolchain)"
 	cargo install cargo-binutils
 
+# The following line gives highlighting on vim
+# vim: set ft=make :
```

Comment:
Keep that one at the end, it gives automatic highlighting in vim (and some other editors) by configuring the file-type to makefile (which is sort of similar enough to justfiles).

I added it back on main as it got deleted on another branch :)